src/main/java/net/sf/jabref/gui/BasePanel.java

```diff
@@ -990,7 +991,8 @@ private void copyKeyAndTitle() {
                     "\\bibtexkey - \\begin{title}\\format[RemoveBrackets]{\\title}\\end{title}\n");
             Layout layout;
             try {
-                layout = new LayoutHelper(sr, Globals.prefs, Globals.journalAbbreviationLoader).getLayoutFromText();
+                layout = new LayoutHelper(sr, LayoutFormatterPreferences.fromPreferences(Globals.prefs),
```

Comment:
Yes, that may make sense. I made an attempt including in JournalAbbreviationPreferences, but that wasn't the correct place it turned out. Will try again.